web3.md

```diff
@@ -293,5 +307,38 @@ module WEB3
     rule <k> #evm_increaseTime => #sendResponse( "result" : Int2String(TS +Int DATA ) ) ... </k>
          <params> [ DATA:Int, .JSONList ] </params>
          <timestamp> ( TS:Int => ( TS +Int DATA ) ) </timestamp>
+
+    syntax KItem ::= "#eth_newBlockFilter"
+ // --------------------------------------
+    rule <k> #eth_newBlockFilter => #sendResponse ( "result": #unparseQuantity( FILTID )) ... </k>
+         <filters>
+           ( .Bag 
+          => <filter>
+               <filterID> FILTID </filterID>
+               <fromBlock> BLOCKNUM </fromBlock>
+               ...
+             </filter>
+           )
+           ...
+         </filters>
+         <number> BLOCKNUM </number>
+         <nextFilterSlot> ( FILTID:Int => FILTID +Int 1 ) </nextFilterSlot>
```

Comment:
Have a case for that filter already existing. Either (i) overwrite it, or (ii) respond with an error message. Do whatever Ganache does in this case.